cli/scorecard/violations.go

```diff
@@ -48,6 +50,32 @@ func addDataFromBucket(config *ScoringConfig, bucket *storage.BucketHandle, obje
 	defer reader.Close()
 
 	scanner := bufio.NewScanner(reader)
+	err = addDataFromScanner(config, scanner)
+	if err != nil {
+		return errors.Wrap(err, "Fetching inventory")
+	}
+
+	return nil
+}
+
+func addDataFromFile(config *ScoringConfig, objectName string) error {
+	reader, err := os.Open(objectName)
+	if err != nil {
```

Comment:
Looks like everything beyond this line is identical in `addDataFromBucket`. Could we consolidate on `addDataFromReader`?

Reply:
addressed in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/314/commits/04fb6f7898a992131c4801e1c2e5d5996eeff114